diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..12ec0bc 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.9.0)
 
 project(rnscreens)
 
+string(APPEND CMAKE_SHARED_LINKER_FLAGS " -Wl,-z,max-page-size=16384")
+
 if(${RNS_NEW_ARCH_ENABLED})
 add_library(rnscreens
     SHARED
